fips202/asm/aarch64/keccak_f1600_x1_scalar_pre_opt.S

```diff
@@ -0,0 +1,934 @@
+        // eor x14, X<tmp>, X<Bmi>, ror #46              // .................................................*...
+        // bic X<tmp>, X<Bma>, X<Bmu>, ror #35           // ..........................................*..........
+        // eor x19, X<tmp>, X<Bmo>, ror #12              // .............................................*.......
+        // bic X<tmp>, X<Bme>, X<Bma>, ror #9            // .....................................*...............
+        // eor x24, X<tmp>, X<Bmu>, ror #44              // ..........................................*..........
+        // bic X<tmp>, X<Bsi>, X<Bse>, ror #48           // ...............................*.....................
+        // eor x5, X<tmp>, X<Bsa>, ror #41               // ...................................*.................
+        // bic X<tmp>, X<Bso>, X<Bsi>, ror #2            // ..............................*......................
+        // eor x10, X<tmp>, X<Bse>, ror #50              // ................................*....................
+        // bic X<tmp>, X<Bsu>, X<Bso>, ror #25           // .....................................*...............
+        // eor x15, X<tmp>, X<Bsi>, ror #27              // .......................................*.............
+        // bic X<tmp>, X<Bsa>, X<Bsu>, ror #60           // ....................................*................
+        // eor x20, X<tmp>, X<Bso>, ror #21              // ....................................*................
+        // bic X<tmp>, X<Bse>, X<Bsa>, ror #57           // .................................*...................
+        // eor x25, X<tmp>, X<Bsu>, ror #53              // ..................................*..................
+        // bic X<tmp>, X<Bbi>, X<Bbe>, ror #63           // .............................................*.......
+        // eor x1, X<Bba>, X<tmp>, ror #21               // ..............................................*......
+        // bic X<tmp>, X<Bbo>, X<Bbi>, ror #42           // ................................................*....
+        // eor x6, X<tmp>, X<Bbe>, ror #41               // ..................................................*..
+        // bic X<tmp>, X<Bbu>, X<Bbo>, ror #57           // ...............................................*.....
+        // eor x11, X<tmp>, X<Bbi>, ror #35              // ................................................*....
+        // bic X<tmp>, X<Bba>, X<Bbu>, ror #50           // ..................................................*..
+        // eor x16, X<tmp>, X<Bbo>, ror #43              // ....................................................*
+        // bic X<tmp>, X<Bbe>, X<Bba>, ror #44           // .................................................*...
+        // eor x21, X<tmp>, X<Bbu>, ror #30              // ...................................................*.
+        // eor x1, x1, X<cur_const>                      // ....................................................*
```

Comment:
I see these lines comment out, I think it's only relevant to SLOTHY dev, can you delete them ?

Reply:
Same